lib/config.ts

```diff
@@ -49,7 +49,7 @@ export type Config = {
   parallelPingFix?: boolean;
   ircStatusNotices?: boolean;
   announceSelfJoin?: boolean;
-  webhooks?: Dictionary<string>;
+  webhooks?: unknown;
```

Comment:
Unknown?  Shouldn't this be a boolean

Reply:
No actually, this is a clever trick to support users with the old record config transparently.

Reply:
As we discussed, this will become a Boolean